components/organisms/Layout.js

```diff
@@ -78,7 +79,11 @@ export default function Layout(props) {
       )}
 
       <main>
-        <div>{props.children}</div>
+        {props.fullscreen ? (
+          props.children
+        ) : (
+          <LayoutContainer>{props.children}</LayoutContainer>
+        )}
```

Comment:
I think my review got deleted but layout and layoutContainer should be applied to _app so they don't have to be added to pages

Reply:
Wasn't quite in the scope of what I was aiming to do, but I've added that change as well.